apps/skeleton-app/src/modules/ApplicationRouting/guards/RedirectToFirstPanelOnUnknownOne.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright LEAV Solutions 2017 until 2023/11/05, Copyright Aristid from 2023/11/06
+// This file is released under LGPL V3
+// License text available at https://www.gnu.org/licenses/lgpl-3.0.txt
+import {FunctionComponent} from 'react';
+import {generatePath, Navigate, useLocation, useOutletContext} from 'react-router-dom';
+import {IApplication, IApplicationMatchingContext, Nullable} from '../types';
+import {routes} from '../routes';
+
+export const RedirectToFirstPanelOnUnknownOne: FunctionComponent<{application: IApplication}> = ({
+    application,
+    children
+}) => {
+    const {search} = useLocation();
+    const {currentPanel} = useOutletContext<Nullable<IApplicationMatchingContext>>();
+
+    return currentPanel == null ? (
```

Comment:
polish: on peut mettre triple égal pour éviter tout de suite les warnings github ?

Reply:
Alors je voudrais bien inclure cette nouvelle syntaxe qui permet de faire plus de choses que de vérifier. C'est moi qui est changé le code pour mettre en warning et plus en erreur. Je trouve ça plus explicite que
```suggestion
    return !!currentPanel ? (
```